kmm/kmm-feed/src/commonMain/kotlin/com/gchristov/newsfeed/kmmfeed/FeedViewModel.kt

```diff
@@ -92,7 +92,7 @@ class FeedViewModel(
             try {
                 val feedUpdate = getSectionedFeedUseCase(
                     pageId = nextPage,
-                    feedQuery = "brexit,fintech",
+                    feedQuery = state.value.searchQuery ?: "brexit,fintech",
```

Comment:
Another way would be making `searchQuery` non-nullable and default it to `brexit,fintech` in the `FeedViewModel` `State`. 
This way we save few null-safe chains. I tried it but got some complaints with nullability in Xcode. It might have just been transient.

What do you think @gchristov? keep it like this or default `searchQuery` in `State` to the current one?